package-lock.json

```
@@ -42956,7 +42956,7 @@
       "dependencies": {
         "strip-ansi": {
           "version": "3.0.1",
-          "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
+          "resolved": "http://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
```

Can you see why this changed? It's minor but this is something I've noticed goes back and fourth in lock file updates. Dependabot PRs change this back to `https` as well. I'll fix it manually here to not block the PR but it's a bit odd.
```suggestion
          "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
```